ReadMe.md

```diff
@@ -25,14 +25,15 @@ If you like or are using this project to learn or start your solution, please gi
    - [JavaScript](https://www.w3schools.com/js/)
    - [NPM](https://docs.npmjs.com/)
    - [Node.js](https://nodejs.org/en/docs/)
-   - ES6/ES7
+   - [ECMAScript](https://www.w3schools.com/js/js_versions.asp)
 
 2. General Development Skills
 
    - Learn GIT, create a few repositories on GitHub, share your code with other people
    - Know HTTP(S) protocol, request methods (GET, POST, PUT, PATCH, DELETE, OPTIONS)
    - Don't be afraid of using Google, [Power Searching with Google](http://www.powersearchingwithgoogle.com/)
    - Read a few books about algorithms and data structures
+   - Clean code (https://www.w3schools.com/js/js_conventions.asp)
```

Comment:
I think clean code is about this page, not w3schools: https://github.com/ryanmcdermott/clean-code-javascript